From 82a3df88cb7aeaeb9a4c2b921d5fa746dc516f0e Mon Sep 17 00:00:00 2001
From: zengxs <xiangsong.zeng@outlook.com>
Date: Fri, 17 Nov 2023 17:29:02 +0800
Subject: [PATCH] Merge upstream

---
 assets/wrapper | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/assets/wrapper b/assets/wrapper
index 6944e90..0668387 100755
--- a/assets/wrapper
+++ b/assets/wrapper
@@ -19,7 +19,7 @@ function failed_pg_upgrade() {
 
 function clean_stale_pids() {
     # cleanup known pid/socket files
-    for x in /opt/gitlab/sv /run ; do
+    for x in /opt/gitlab/sv /run /var/opt/gitlab ; do
         # find
         #  - any (s)ocket or regular (f)ile
         #  - by the name of "*.pid" or "socket.?"
-- 
2.40.0

